adapter/include/hermes/adapter/stdio.h

```diff
@@ -69,4 +69,12 @@ HERMES_FORWARD_DECL(fsetpos, int, (FILE * stream, const fpos_t *pos));
 HERMES_FORWARD_DECL(fsetpos64, int, (FILE * stream, const fpos64_t *pos));
 HERMES_FORWARD_DECL(rewind, void, (FILE * stream));
 
+/**
+ * MPI functions declarations
+ */
+HERMES_FORWARD_DECL(PMPI_Init, int, (int *argc, char ***argv));
+HERMES_FORWARD_DECL(PMPI_Finalize, int, (void));
```

Comment:
It looks like we're not using the `PMPI_` variations. Are you leaving these declarations here because you plan to use them in the future, or can they be removed?

Reply:
FIXED